### DISCUSSION TITLE
Add a script to the BG matchmaking system that checks the spec of incoming players to have an even distribution of healers.

### DISCUSSION THREAD

**Discussion by Eledor-collab** [chromiecraft/chromiecraft#7037] [General]
I'm not a coding nerd, so, I have no clue if it's feasible. But I think this could be a good idea to implement a script that checks ppl specs before assigning them into a team. 

A guy joining the queue is checked to see if he is a healer, and if he is, he is sent to the team with the fewest healers. The goal would be to avoid having 3 healers on one side of a team, while the other side just waits to be trashed in GY.

**Comment by heyitsbench** [comment]
What about dual spec? Queue in DPS spec so the system believes you to be DPS and switch once you get matched.

**Reply by Eledor-collab** [reply]
True! that's a very valid point. I guess the script should start working after the game has started. That way, only new incoming healers will be distributed. 

And yeah, you still have the risk of starting the game with uneven number of healers. But just distributing incoming new healers could improve players quality of life during BGs.
